cli.js

```diff
@@ -23,10 +24,23 @@ function parse(data) {
   }
 }
 
+dns.lookup('www.google.com', err => {
+  if (err && err.code === 'ENOTFOUND') {
+    spinner.stop();
+    console.log(chalk.bold.red('É preciso estar conectado com a internet para validar o domínio.'));
+    process.exit(1);
+  }
+});
+
 if (!url || url.length < 2) {
   console.log(chalk.red('Por favor, digite uma url válida.'));
   process.exit(1);
 } else {
+  if (url.indexOf('.br') === -1) {
+    console.log(chalk.red('A url informada deve possuir a extensão .br'));
```

Comment:
Apesar de Registro.br estar claro de que você está registrando um domínio `.br`, não pode confundir quem não conhece?

Se for mantê-lo, sugiro que atualizemos a mensagem do loading https://github.com/ftonato/registrobr/blob/master/cli.js#L30 para incluir a extensão caso o usuário não coloque. Pois gera uma percepção de que eu estou pesquisando exatamente aquele domínio que foi informado.